Fix issue 34, add CONTRIBUTING.md

alexxa opened the pull request:
In this PR: 
- add CONTRIBUTING.md file
- correct relevant docs files 

@jscotka, @phracek @bkabrda please review CONTRIBUTING.md whether you are ok with its context, or I missed smth, and let me know whether you agree to follow those guidelines. Thanks.

phracek commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,139 @@
+
+Before creating bug reports, please check a [list of known issues](https://github.com/fedora-modularity/meta-test-family/issues) to see if the problem has already been reported.
+
+If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/fedora-modularity/meta-test-family/issues/new). Be sure to include a **descriptive title and clear description** and please include **as many details as possible** to help maintainers reproduce the issue and resolve it faster. If possible, add a **code sample** or an **executable test case** demonst ...
```
Can you please add also info about version? We have several releases and it would be easy to find out what was changed since last tag.

phracek commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,139 @@
+
+### Submitting changes
+
+To submitt changes, please send a [GitHub Pull Request](https://github.com/fedora-modularity/meta-test-family/pull/new/devel). Before submitting a PR, please read the [Styleguides](#styleguides) to know more about coding conventions used in this project. Also we will appreciate if you check your code with pylint and pyflakes. Always create a new branch for each pull request to avoid intertwinglin ...
```
Please mention last sentence on separate row. It would be more clear. I have missed it by reading whole paragraph.

phracek commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,139 @@
+
+  * Reference issues and pull requests liberally after the first line
+  * When only changing documentation, include `[ci skip]` in the commit description
+  * Consider starting the commit message with an applicable emoji:
```
Any reason why to use emoji in commit messages?

jscotka commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,139 @@
+      * :arrow_up: `:arrow_up:` when upgrading dependencies
+      * :arrow_down: `:arrow_down:` when downgrading dependencies
+      * :shirt: `:shirt:` when removing linter warnings
+
```
I've never used these emoji, just RFE: as mark, that it is not bug fix. or WIP:, as this is not ready to merge. it is work in progress (CI is started based on PR, so that it is important)
But I'm not against. If you find this usefull. And I would like to add also these Two there,

alexxa commented:
@phracek Requested changes we added within commit 5b584ff. Also please notice I updated the list of emoji. Though I'm not sure whether it's better to use them in the beginning of the commit message or in the end,

@phracek, @jscotka First, using emoji is completely optional, as this file says - these are not rules, but guidelines. We, as developers, routinely look at large amounts of text: code, commit messages, documentation, etc and emoji inherently stand out in what is normally a wall of text. It’s far easier to pick an emoji out of a text than a random string. Emoji are like labels. Also, even if there is no actual productivity gain - using emoji is just plain fun. Check how emoji in commit messages look within my two last commits [here](https://github.com/fedora-modularity/meta-test-family/commits/docs).
